cli/open.go

```diff
@@ -327,6 +331,428 @@ func resolveAgentAbsPath(workingDirectory, relOrAbsPath, agentOS string, local b
 	}
 }
 
+const (
+	cursorDesktopName   = "Cursor Desktop"
+	windsurfDesktopName = "Windsurf Desktop"
+	fleetDesktopName    = "JetBrains Fleet"
+	zedDesktopName      = "Zed"
+)
+
+// openCursor implements the "coder open cursor" command which opens a workspace in Cursor IDE
+func (r *RootCmd) openCursor() *serpent.Command {
+	return r.createIDECommand(
```

Comment:
```suggestion
	return r.createGenericVSCodeIDECommand(
```
I like that this was turned into shared code, naming however could be a bit clearer here.

I suppose it begs to question, though, why isn't the `VSCode` command using the shared code?

Reply:
I think I asked it to not change how the Vscode works significantly.